src/h5web/visualizations/heatmap/ColorBar.tsx

```diff
@@ -1,29 +1,28 @@
 import React from 'react';
 import { AxisRight } from '@vx/axis';
 import { useMeasure } from 'react-use';
-import shallow from 'zustand/shallow';
 import { adaptedNumTicks } from '../shared/utils';
 import { useInterpolator } from './hooks';
-import { useHeatmapConfig } from './config';
 import styles from './HeatmapVis.module.css';
 import { generateCSSLinearGradient } from './utils';
-import { SCALE_FUNCTIONS } from '../shared/models';
-
-function ColorBar(): JSX.Element {
-  const [dataDomain, customDomain, scaleType] = useHeatmapConfig(
-    (state) => [state.dataDomain, state.customDomain, state.scaleType],
-    shallow
-  );
+import { SCALE_FUNCTIONS, ScaleType, Domain } from '../shared/models';
 
+function ColorBar({
+  domain,
+  scaleType,
+}: {
+  domain: Domain | undefined;
+  scaleType: ScaleType;
+}): JSX.Element {
```

Comment:
Formatting can be fixed with:

- `props` + destructuring inside the function
- `Props` interface

Reply:
Oops, I wrote that for quick prototyping and let this slide :sweat: